src/package.xml

```diff
@@ -1479,19 +1479,17 @@
         <name>CustomLabel</name>
     </types>
     <types>
-        <name>CustomLabel</name>
+        <members>CustomLabels</members>
+        <name>CustomLabels</name>
     </types>
     <types>
-        <members>Account</members>
```

Comment:
were all these standard objects supposed to be deleted from package.xml?  did you remember to run ant updatePackageXml?